packages/sdk/src/types/FrameTypes.ts

```diff
@@ -274,3 +276,84 @@ export enum UpdateZIndexMethod {
     bringForward = 'bringForward',
     sendBackward = 'sendBackward',
 }
+
+export enum FrameAnchorType {
+    relative = 'relative',
+    start = 'start',
+    end = 'end',
+    startAndEnd = 'startAndEnd',
+    center = 'center',
+}
+
+export enum AnchorTargetType {
+    page = 'page',
+    frame = 'frame',
+}
+
+export enum AnchorTargetEdgeType {
+    start = 'start',
+    end = 'end',
+    center = 'center',
+}
+
+export class PageAnchorTarget {
+    type = AnchorTargetType.page;
+}
+
+export class FrameAnchorTarget {
+    id: Id;
+    edge: AnchorTargetEdgeType;
+    type = AnchorTargetType.frame;
+
+    constructor(id: Id, edge: AnchorTargetEdgeType) {
+        (this.id = id), (this.edge = edge);
+    }
```

Comment:
why is this relevant?

Reply:
Could be a misunderstanding of typescript on my side. I made it a class just so that I could create a page anchor target without having to define the type, which is already known in the type definition (& I assume is more of a constraint here). I basically wanted to do `new PageAnchorTarget()`;

Reply:
@alexandraFlavia9 @abdelhalimkhouas WDYT?

Reply:
I would go with the simpler approach and just do
````

 export interface FrameAnchorTarget {
    id: Id;
    edge: AnchorTargetEdgeType;
    type = AnchorTargetType.frame;
}
````
I see u r using `new FrameAnchorTarget` in the tests, if there is no other use I suggest to use an interface instead of a class.

Reply:
After a little chat w/ Halim we're gonna keep it as-is because this model, while it will be used from engine to SDK, it will **also** be used as an object from SDK to engine via the `setHorizontalAnchor` & `setVerticalAnchor` methods.